components/charity/CharityCard.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <v-row
+  >
+    <v-col
+      cols="12"
+      md="auto"
+      class="py-0 pl-0"
+    >
+      <v-img
+        width="250"
+        height="280"
+        class="charity-card-img"
+        :src="require('@/assets/images/slideshow1.jpg')"
+      />
+    </v-col>
+    <v-col
+      class="pa-0"
+    >
+      <v-row>
+        <v-col
+          class="py-0"
+        >
+          <h2>{{ gatherings[id].name }}, {{ gatherings[id].age }} лет</h2>
+          <p>{{ gatherings[id].problem }}</p>
+        </v-col>
+        <v-col
+          class="py-0"
+          cols="auto"
+        >
+          <v-progress-circular
+            color="primary"
+            :value="gatherings[id].collectedPercent"
+            size="70"
+            rotate="-90"
+          >
+            <b class="percent-text">{{ gatherings[id].collectedPercent }}%</b>
+          </v-progress-circular>
+        </v-col>
+      </v-row>
+      <p class="description mt-3">
+        {{ gatherings[id].text }}
+      </p>
+      <v-row justify="space-between" class="px-3 mt-8">
+        <v-col
+          cols="12"
+          md="8"
+          class="pa-0"
+        >
+          <v-row
+            justify="space-between"
+            class="px-3"
+          >
+            <div>
+              <v-card-text class="pa-0 ma-0 progress-text">
+                {{ $t('charity.gathering.charityCard.collected') }}
+              </v-card-text>
+              <b>{{ gatherings[id].collected }} ₽</b>
```

Comment:
Use the `rubles` filter
```suggestion
              <b>{{ gatherings[id].collected | rubles }}</b>
```
Same for the other usages